### PR TITLE
9587 - Practitioner Documentations

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: web-api/src/app.js [hunk 1]
```diff
@@ -263,6 +266,12 @@ const {
 const {
   getPractitionerByBarNumberLambda,
 } = require('./practitioners/getPractitionerByBarNumberLambda');
+const {
+  getPractitionerDocumentDownloadUrlLambda,
+} = require('./practitioners/getPractitionerDocumentDownloadUrlLambda');
+const {
+  getPractitionerDocumentsLambda,
+} = require('./practitioners/getPractitionerDocumentsLambda');
```
Comment (TomElliottFlexion) [inline web-api/src/app.js:274]: Any idea why we're sorting single and multi-line imports separately?

Comment (codyseibert) [reply web-api/src/app.js:274]: I'm not sure.  All we did was fork that existing sort plugin and fix a small syntax error.

**Review by TomElliottFlexion — APPROVED** [review]

File: shared/src/business/useCases/practitioners/getPractitionerDocumentDownloadUrlInteractor.test.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    await expect(
+      getPractitionerDocumentDownloadUrlInteractor(applicationContext, {
+        barNumber: mockPractitioner.barNumber,
+        practitionerDocumentFileId:
+          mockDocumentMetadata.practitionerDocumentFileId,
+      }),
+    ).rejects.toThrow(UnauthorizedError);
+  });
+
+  it('should not throw an unauthorized error when the user does not have permission to download the practitioner documentation file', async () => {
```
Comment (mmarcotte) [inline shared/src/business/useCases/practitioners/getPractitionerDocumentDownloadUrlInteractor.test.ts]: can you rephrase this `it` statement? I think it's meant to be:

> should not throw an unauthorized error when the user has permission to download the practitioner documentation file

or perhaps better

> successfully download the document when the user has permission to download the practitioner documentation file

File: web-client/src/presenter/actions/createPractitionerDocumentAction.test.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import { createPractitionerDocumentAction } from './createPractitionerDocumentAction';
+import { presenter } from '../presenter-mock';
+import { runAction } from 'cerebral/test';
+
+describe('createPractitionerDocumentAction', () => {
+  beforeAll(() => {
+    presenter.providers.applicationContext = applicationContext;
+  });
+
+  it('should try to upload the file on state to s3', async () => {
```
Comment (mmarcotte) [inline web-client/src/presenter/actions/createPractitionerDocumentAction.test.js]: super nitpicky, but i don't think it `to s3` is right here as really these could be getting stored anywhere, and nothing below is really testing that aspect.

File: web-client/src/presenter/computeds/practitionerDocumentationFormHelper.test.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+            permissions: {
+              UPLOAD_PRACTITIONER_DOCUMENT: true,
+            },
+          },
+        },
+      );
+      expect(isCertificateOfGoodStanding).toBe(true);
+    });
+
+    it('should return false for isCertificateOfGoodStanding if documentationCategory equals "Certificate of Good Standing"', () => {
```
Comment (mmarcotte) [inline web-client/src/presenter/computeds/practitionerDocumentationFormHelper.test.js:34]: i think this it statement also needs to be amended.

**Review by mmarcotte — APPROVED** [review]

File: shared/src/business/useCases/practitioners/getPractitionerDocumentDownloadUrlInteractor.test.ts (hunk 2 above)
Comment (rgoshen) [reply shared/src/business/useCases/practitioners/getPractitionerDocumentDownloadUrlInteractor.test.ts]: rephrased to 'should not throw an unauthorized error when the user has permission to download the practitioner documentation file'

File: web-client/src/presenter/actions/createPractitionerDocumentAction.test.js (hunk 3 above)
Comment (rgoshen) [reply web-client/src/presenter/actions/createPractitionerDocumentAction.test.js]: rephrase to 'should try to upload the file on state to persist the new practitioner document'

File: web-client/src/presenter/computeds/practitionerDocumentationFormHelper.test.js (hunk 4 above)
Comment (rgoshen) [reply web-client/src/presenter/computeds/practitionerDocumentationFormHelper.test.js:34]: rephrased to 'should return false for isCertificateOfGoodStanding if categoryType is undefined'
